gcp/modules/locust/client.rb

```diff
@@ -68,5 +71,10 @@ def process_locust_result(locust_stats_file, locust_distribution_file, app_name)
     }
   end
 
-  metric_service_client.create_time_series(formatted_name, time_series)
+  begin
+    metric_service_client.create_time_series(formatted_name, time_series)
+  rescue Google::Gax::RetryError
+    puts "[ERROR]: Error while submitting metrics to Stackdriver (Google::Gax::RetryError)."
+    exit 120
```

Comment:
This custom exit code needed so we can later use it in the shell code to retry only in case of `Google::Gax::RetryError` errors.